apps/landingPage/components/Headline/GetStartedInfo.js

```diff
@@ -45,16 +58,23 @@ const ContainerLeftText = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: space-around;
-  height: 320px;
-  width: 80vw;
+  width: 85vw;
   padding-left: 0;
   padding-top: 4vh;
   align-items: flex-start;
-
   @media (min-width: ${BREAKPOINTS.BIG_MOBILE}px) and (max-width: ${BREAKPOINTS.TABLET}px) {
     align-items: center;
   }
+  @media (min-width: ${BREAKPOINTS.BIG_MOBILE}px) and (max-width: ${BREAKPOINTS.BIG_TABLET}px) {
+    align-items: center;
+  }
   @media (min-width: ${BREAKPOINTS.TABLET}px) {
+    flex-direction: 'row';
```

Comment:
```suggestion
    flex-direction: row;
```

But probably it can be removed